metadata-models/src/main/pegasus/com/linkedin/dataJob/azkaban/AzkabanClusterInfo.pdl

```diff
@@ -0,0 +1,12 @@
+namespace com.linkedin.dataJob.azkaban
+
+/**
+ * Azkaban cluster information which includes azkaban cluster name
+ */
+record AzkabanClusterInfo {
```

Comment:
A typeref might be more suitable unless you plan to expand this record with extra info.